src/Data/V/Linear/Internal/V.hs-boot

```diff
@@ -0,0 +1,53 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FunctionalDependencies #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE LinearTypes #-}
+{-# LANGUAGE MagicHash #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE StandaloneKindSignatures #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE UnboxedTuples #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+
+module Data.V.Linear.Internal.V
+  ( V (..),
+    consume,
+    -- consumeV,
+    fmap,
+    pure,
+    (<*>),
+    next#,
+    next,
+    -- split,
+    Elim (..),
+    cons,
+    -- fromReplicator,
+    -- dupV,
+  )
```

Comment:
You don't have to write everything in the hs-boot file. Just whatever you will need in your loop. (I'm still unhappy about this, I very much expect that it doesn't survive until the end of the week)